nimble/host/syscfg.yml

```diff
@@ -331,6 +331,19 @@ syscfg.defs:
         description: >
             MTU used for EATT channels.
         value: 128
+    BLE_EATT_AUTO_CONNECT:
+        description: >
+            Enable auto connect for EATT
+        value: 1
+        restrictions:
+            - BLE_L2CAP_ENHANCED_COC
+    BLE_EATT_CHAN_PER_CONN:
+        description: >
+            Maximum number of supported EATT channels per connection
+        value: 5
```

Comment:
same here,

I'm not sure if there is support arithmetic in here, if not I'd suggest to keep it simple for now and default to 0 when eatt is disabled and 1 when enabled,    default can be tuned later, while target can set what value is needed

Reply:
Done